src/Chapter19.Tests/Listing19.06.Tests.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace AddisonWesley.Michaelis.EssentialCSharp.Chapter19.Listing19_06.Tests
+{
+    [TestClass]
+    public class ProgramTests
+    {
+        [TestMethod]
+        public void MainTest()
+        {
+            const string expected = @"ERROR: Operation is not valid due to the current state of the object.";
+
+            IntelliTect.TestTools.Console.ConsoleAssert.Expect(
+                expected, Program.Main);
```

Comment:
Is this an exception that we want to catch or an error we are printing to the console?